_draft-tests/SC2-4-2-title.md

```diff
@@ -0,0 +1,50 @@
+---
+rule_id: SC2-4-2-title
+test_case_template: default
+success_criterion:
+- 2.4.2
+---
+
+### Passed
+
+```html
+<html>
+  <title>This page has a title</title>
+</html>
+```
+
+```html
+<html>
+  <title>This page gives a title to an iframe</title>
+  <iframe src="page-without-title.html"></iframe>
```

Comment:
Unclear what "This page gives a title to an iframe" means.

Reply:
Done, created the pages.